mono/io-layer/sockets.c

```diff
@@ -664,55 +664,6 @@ WSAIoctl (guint32 fd, gint32 command,
 		return(SOCKET_ERROR);
 	}
 
-	if (command == SIO_KEEPALIVE_VALS) {
```

Comment:
How did you determine this to be dead code?

It looks like this is used by the BCL.

```
git grep 0x98000004
mcs/class/referencesource/System/net/System/Net/Sockets/IOControlCode.cs:        KeepAliveValues = 0x98000004,
mono/io-layer/sockets.h:#define SIO_KEEPALIVE_VALS 0x98000004
```

Then
```
git grep KeepAliveValues
mcs/class/System/System.Net/ServicePoint.cs:                    socket.IOControl (IOControlCode.KeepAliveValues, bytes, null);
mcs/class/referencesource/System/net/System/Net/ServicePoint.cs:                                IOControlCode.KeepAliveValues,
mcs/class/referencesource/System/net/System/Net/Sockets/IOControlCode.cs:        KeepAliveValues = 0x98000004,
```

Which suggests at least two places could be potentially using this.